src/overlays/actors/ovl_En_Gs/z_en_gs.c

```diff
@@ -426,7 +426,7 @@ void func_80A4F13C(EnGs* this, PlayState* play) {
         }
     }
     if (this->unk_19F == 3) {
-        this->unk_1A0[0].y += 0x4000;
+        this->unk_1A0[0].y += DEG_TO_BINANG2(90.0f);
```

Comment:
since this is a literal it can use `DEG_TO_BINANG` can't it?

Reply:
This is a float literal on N64 and an int literal on GC

Reply:
Oh the left hand side is a s16 nvm I see